modules/virt-creating-vm-instancetype.adoc

```diff
@@ -6,30 +6,25 @@
 [id="virt-creating-vm-instancetype_{context}"]
 = Creating a virtual machine from an instance type
 
-You can use common instance types to install and edit customized instance types and preferences to create a virtual machine (VM).
+You create a virtual machine (VM) from an instance type by using the {product-title} web console.
 
 :FeatureName: Creating a virtual machine from an instance type
 include::snippets/technology-preview.adoc[]
 
 .Procedure
 
-. In the web console, navigate to *Virtualization* -> *Catalog* -> *InstanceTypes*.
-
-. Select a volume to boot from.
+. In the web console, navigate to *Virtualization* -> *Catalog* and click the *InstanceTypes* tab.
 
+. Select a bootable volume.
 +
 [NOTE]
 ====
-The volumes list shows only volumes that exist in the `openshift-virtualization-os-images` namespace and have the `instancetype.kubevirt.io/default-preference` label.
+The volume table displays only volumes that exist in the `openshift-virtualization-os-images` namespace and have the `instancetype.kubevirt.io/default-preference` label.
```

Comment:
Nitpicky minimalism suggestion: 

```
The volume table only lists volumes in the `openshift-virtualization-os-images` namespace that have the `instancetype.kubevirt.io/default-preference` label.
```

Reply:
Much better than what the 'other' writer came up with.